### PR TITLE
Create e2e tests for executive summary - default values

### LINKED ISSUE #3435: Add e2e test for checking the Executive Summary on an empty APD
### Description and related issues -
Using the empty APD created in #3282, write a test that checks the default values in the Executive Summary page. 

### Test cases needed...

- [x] should have default values in the Executive Summary
    - [x] login as State Staff, click 'HITECH IAPD' and navigate to the Executive Summary
    - [x] verify the default FFYs show up in the header
    - [x] should have default values in the Activities Summary
        - [x] verify the Activity 1: Program Administration exists
        - [x] verify the start and end dates 'Date not specified' 
        - [x] verify the Total Cost of Activity 1 is $0
        - [x] verify the Total Computable Medicaid cost is $0 with a federal share of $0
            - [x] verify that each FFY costs $0 with a Total Computable Medicaid Cost of $0 and federal share of $0
        - [x] verify that clicking edit next to activity 1 takes you to the activity overview page for activity 1
    - [x] should have default values in the Total Cost section
        - [x] verify the default FFYs show up in Federal Fiscal Years Requested
        - [x] verify the Total Computable Medicaid Cost is $0 with a federal share of $0
        - [x] verify the Total Funding Request is $0
        - [x] verify that each FFY costs $0 with a Total Computable Medicaid Cost of $0 and federal share of $0
    - [x] check default values in HIT + HIE and MMIS table
        - [x] verify that each FFY row is $0  and Total Computable Medicaid Cost column is $0
        - [x] verify that the total row is $0 and the Total Computable Medicaid Cost column is 0$
- [x] Make sure to run this test before any of the tests that fill out the APD
    - [x] You can control the order of the tests by adding a number in the beginning of the file name.

### PR DESCRIPTION
**Pull request by chris-miraflor**
resolves #3435 

**Description-**
Adds e2e tests for the executive summary page for default values, before activities have been added/edited in an APD.

**This pull request changes...**

- Added test specification for executive summary
- Added page object for interacting with executive summary page

**This pull request also touches…**

- A refactor of older e2e tests is included with this PR
- Cypress commands for navigating using the nav sidebar were added

**This pull request was tested in the follow ways…**

- Creating a new 

**Steps to manually verify this change...**

1. Open cypress using the `cypress-setup.sh` script with either `./cypress-setup.sh open` or `./cypress-setup.sh`
2. If opening cypress rather than running all tests, run the first test `01-apd-overview.spec.js` to ensure an empty APD is created, then run `07-default-executive-summary.spec.js`
3. Verify that all tests pass

### This pull request is ready to review when...

- [X] Automated tests are updated (and all tests are passing)
- [ ] The change has been documented
- [ ] Associated OpenAPI documentation has been updated
- [ ] Changelog is updated as appropriate
- [ ] The experience passes a basic manual accessibility audit (keyboard nav, screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when…

- [X] Code has been reviewed by someone other than the original author
- [ ] QA has verified the accessibility and functionality related to the change
- [ ] Design has approved the experience
- [ ] Product has approved the experience

### CODE REVIEW

File: integrationTests/cypress/support/commands.js [hunk 1]
```diff
@@ -124,8 +124,60 @@ Cypress.Commands.add('ignoreTinyMceError', () => {
...
+  cy.get('.ds-c-vertical-nav__label--parent')
+    .contains(/Proposed Budget/i)
+    .click();
+  // Click on nav submenu button
+  cy.get('a.ds-c-vertical-nav__label')
+    .contains(/Proposed Budget/i)
+    .click();
+});
+
+Cypress.Commands.add('goToAssuranceCompliance', () => {
```
Comment (thetif) [inline integrationTests/cypress/support/commands.js:166]: when you are calling this you call it goToAssurancesCompliance so it's causing the tests to fail

**Review by thetif — COMMENTED** [review]
The code is looking really good, but I think that it's grabbing the wrong APD for the default tests and they are failing. Could you take a look at it again and make sure that it's testing the correct one?

**Review by thetif — APPROVED** [review]
Looks good!
